assets/css/style.css

```diff
@@ -18,9 +18,11 @@
   padding: 0;
   margin: 0;
   box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
 }
 
 body {
+  width: 100vw;
```

Comment:
What is the purpose of this?

Reply:
It prevents the body from moving in the x-axis